tests/GameTests.cpp

```diff
@@ -11,3 +13,19 @@ TEST_F(GameTests, AddFrames)
     game.addFrames("Someone:32|4");
 }
 
+TEST_F(GameTests, conversion_to_int)
+{
+    Game game;
+    auto result1 = game.conversion_to_int("X");
+    ASSERT_EQ(result1, 10);
+    auto result2 =game.conversion_to_int("-");
```

Comment:
Możesz ponownie używać tej samej zmiennej we wszystkich sprawdzeniach, albo nawet ją pominąć. Taki kod będzie bardziej optymalny.
```cpp
ASSERT_EQ(game.conversion_to_int("X"), 10);
ASSERT_EQ(game.conversion_to_int("-"), 0);
ASSERT_EQ(game.conversion_to_int("4"), 4);
```